### DISCUSSION TITLE
How to override .gitignore setting?

### DISCUSSION THREAD

**Discussion by AtomicNess123** [BurntSushi/ripgrep#1800] [General] [answered]
Basically that, in order to be able to search in the ignored folders and files, without having to edit the .gitignore file. Thanks in advance!

**Comment by BurntSushi** [comment, marked as answer]
As mentioned before, please consult the guide: https://github.com/BurntSushi/ripgrep/blob/master/GUIDE.md#automatic-filtering

If there's something there you don't understand, then please ask a more specific question.

**Comment by AtomicNess123** [comment]
--no-ignore!
